wLabs/LabSetup.md

```diff
@@ -0,0 +1,595 @@
+---
+title: Setting up the Lab Environment
+layout: custom
+keywords: [lab, guide, setup, start, FinCube, data cells]
+description: This page details the necessary requirements and steps to set up a local environment that can accommodate a user following the Lab Guide.
+---
+* * *
+
+## Overview
+
+The [Lab Guide](/wLabs/lab.html) provides a convenient way for users to become familiar with Interject. The Guide walks through several individual walkthroughs on how to modify and create reports, drilling to data, and exporting reports. In order for a user to follow along, it is necessary to first set up a local environment that can accommodate this Lab Guide. This page details the necessary requirements and steps to do this.
```

Comment:
```suggestion
The [Lab Guide](/wLabs/lab.html) provides a convenient way for users to become familiar with Interject. The Guide has several walkthroughs on how to modify and create reports, drill to data, and export reports. In order for a user to follow along, it is necessary to first set up a local environment that can accommodate this Lab Guide. This page details the necessary requirements and steps to do this.
```

- walkthrough was mentioned twice... could be more succinct
- rephrase modify\create\drilling\exporting to the same verb style